### DISCUSSION TITLE
Scale object and display new size in meters

### DISCUSSION THREAD

**Discussion by robertmajhen** [google/model-viewer#4160] [Q&A]
Annotation sample with displayed initial measurement of the object is great. 
How to display new measures after scaling the object (pinch in AR) or get the info of new dimensions of scaled object?

**Comment by elalish** [comment]
Hmm, I feel like https://modelviewer.dev/docs/index.html#entrydocs-loading-methods-getDimensions should take that kind of scaling into account, but I don't think it currently does. Interested in trying a PR to fix that?

**Reply by robertmajhen** [reply]
Unfortunately getDimensions does not do that ... it returns initial values. PR ... Sure .. why not.
